packages/gatsby/src/schema/types/built-in-types.ts

```diff
@@ -76,13 +76,15 @@ const site = `
   type Site implements Node @infer {
     buildTime: Date @dateformat
     siteMetadata: SiteSiteMetadata
+    head: Boolean
   }
 `
 
 const siteSiteMetadata = `
   type SiteSiteMetadata {
     title: String
     description: String
+    image: String
```

Comment:
Needed to include this in the default GraphQL schema so `head/index.js` could query for it even if users don't define it.